supervisor/daemon/daemon.go

```diff
@@ -71,6 +72,8 @@ func (d *Daemon) dialog(conn io.ReadWriter) {
 		cmd := strings.Split(string(line), " ")
 		op := strings.ToLower(cmd[0])
 		switch op {
+		case commandVersion:
```

Comment:
We do not pass any ldflags for supervisor binary when building. Is it working for your?

Reply:
we do in `buildBinaryFor`